splash/qtrender_lua.py

```diff
@@ -1273,13 +1292,85 @@ def clear(self):
 
 class _ExposedElement(BaseExposedObject):
     FLAG_EXCEPTIONS = [DOMError]
-    _attribute_whitelist = ['id']
+    _attribute_whitelist = ['']
+    HTMLELEMENT_PROPERTIES = [
+        ('accessKey', False),
+        ('accessKeyLabel', True),
+        ('contentEditable', False),
+        ('isContentEditable', True),
+    ]
+    ELEMENT_PROPERTIES = [
+        ('attributes', True), # NamedNodeMap
+        ('classList', True), # DOMTokenList
+        ('className', False),
+        ('clientHeight', True),
+        ('clientLeft', True),
+        ('clientTop', True),
+        ('clientWidth', True),
+        ('id', False),
+        ('innerHTML', False),
+        ('localeName', True),
+        ('namespaceURI', True),
+        ('nextElementSibling', True),
+        ('outerHTML', False),
+        ('prefix', True),
+        ('previousElementSibling', True),
+        ('scrollHeight', True),
+        ('scrollLeft', False),
+        ('scrollTop', False),
+        ('scrollWidth', True),
+        ('tabStop', False),
+        ('tagName', True),
+    ]
+    NODE_PROPERTIES = [
+        ('baseURI', True),
+        # ('childNodes', True),
+        ('firstChild', True),
+        ('lastChild', True),
+        ('nextSibling', True),
+        ('nodeType', True),
+        ('nodeValue', True),
+        ('ownerDocument', True),
+        ('parentNode', True),
+        ('parentElement', True),
+        ('previousSibling', True),
+        ('rootNode', True),
+        ('textContent', False),
+
+
+    ]
 
     def __init__(self, lua, exceptions, element, splash):
         self.element = element
         self.splash = splash
         super(_ExposedElement, self).__init__(lua, exceptions)
 
+    @staticmethod
+    def init_properties():
```

Comment:
You can make it
```
@classmethod
def init_properties(cls):
    ....
```
and refer to the class as `cls` if that's more convenient (but i don't insist)